### ISSUE TITLE
-[__NSDictionaryM setObject:forKey:]: object cannot be nil (key: OriginURL)' was thrown while invoking set on target RNCookieManagerIOS

### ISSUE THREAD

**Issue by klaevv** [react-native-community/cookies#18]
# Bug

I'm moving a cookie from `WKHTTPCookieStore`to `NSHTTPCookieStorage` by using the `useWebKit` flag.

```
  CookieManager.getAll(true)
	  .then((res) => {
      if (res.loginCookie) {
        const loginCookie = {
          ...res.loginCookie,
          origin: 'some origin',
          version: '1',
          expiration: '2035-05-30T12:30:00.00-05:00'
        }
        CookieManager.set(loginCookie, false)
      }
  })
```

I have to set dummy values to origin, version and expiration because react-native-cookies doesn't pass these values from `getAll`. 

## Environment info

React native info output:

```bash
System:
    OS: macOS Mojave 10.14.6
    CPU: (8) x64 Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
    Memory: 809.59 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 10.16.0 - ~/.nvm/versions/node/v10.16.0/bin/node
    Yarn: 1.17.3 - /usr/local/bin/yarn
    npm: 6.13.0 - ~/.nvm/versions/node/v10.16.0/bin/npm
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
    Android SDK:
      API Levels: 21, 23, 24, 25, 26, 27, 28
      Build Tools: 23.0.1, 25.0.2, 26.0.1, 27.0.3, 28.0.0, 28.0.1, 28.0.2, 28.0.3
      System Images: android-23 | Google APIs Intel x86 Atom, android-24 | Google Play Intel x86 Atom, android-27 | Google APIs Intel x86 Atom, android-27 | Google Play Intel x86 Atom, android-28 | Google APIs Intel x86 Atom, android-28 | Google Play Intel x86 Atom, android-29 | Google APIs Intel x86 Atom
  IDEs:
    Android Studio: 3.4 AI-183.6156.11.34.5692245
    Xcode: 11.3.1/11C504 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: 0.61.4 => 0.61.4 
  npmGlobalPackages:
    react-native-cli: 2.0.1

```

Library version: 1.1.0

## Steps To Reproduce

The error occurs if the dummy values are not passed to origin, version and expiration.

Describe what you expected to happen:

1. Cookie is set to NSHTTPCookieStorage.
2. No errors

## Reproducible sample code

```
  CookieManager.getAll(true)
	  .then((res) => {
      if (res.loginCookie) {
        CookieManager.set(res.loginCookie, false)
      }
  })
```

**Comment by safaiyeh** [comment]
Thank you for creating this issue. I'll look into this sometime this week.

**Comment by safaiyeh** [comment]
Released with version `2.0.3`
